osu.Game/Screens/Select/BeatmapClearScoresDialog.cs

```diff
@@ -39,6 +39,28 @@ public BeatmapClearScoresDialog(BeatmapInfo beatmap, Action onCompletion)
             };
         }
 
+        public BeatmapClearScoresDialog(ScoreInfo score, Action onCompletion)
+        {
+            string accuracy = string.Format(score?.Accuracy % 1 == 0 ? @"{0:P0}" : @"{0:P2}", score?.Accuracy);
+
+            BodyText = $@"{score?.Beatmap?.Metadata?.Artist} - {score?.Beatmap?.Metadata?.Title} {Environment.NewLine} {score?.User} - Rank: {score?.Rank} - Max Combo: {score?.MaxCombo} - {accuracy} - {score?.Date.Date.ToShortDateString()}";
+
+            Icon = FontAwesome.Solid.Eraser;
+            HeaderText = @"Clearing this local score. Are you sure?";
+            Buttons = new PopupDialogButton[]
+            {
+                new PopupDialogOkButton
+                {
+                    Text = @"Yes. Please.",
+                    Action = (() => scoreManager.Delete(score)) + onCompletion
```

Comment:
While this isn't going to be as expensive as bulk removal of scores for one beatmap, there could still be some DB work involved, which means I/O. We might want this to be asynchronous mirroring how the other one is done.

Reply:
I did have tried to do it asynchronously, but the score board does not refresh once the delete is completed.

Will look more into it and fix it.

Reply:
no need for async for now. keep it simple.

Reply:
ok sounds good for now.